src/SonarScanner.MSBuild.Tasks/Resources.resx

```diff
@@ -145,7 +145,7 @@ Error: {1}</value>
     <value>Failed to resolve path relative to project file. Path: {0}</value>
   </data>
   <data name="WPIF_MissingOrInvalidProjectGuid" xml:space="preserve">
-    <value>The project does not have a valid ProjectGuid. Analysis results for this project will not be uploaded. Project file: {0}</value>
+    <value>Sonar: The project does not have a valid ProjectGuid. Analysis results for this project will not be uploaded. Project file: {0}</value>
```

Comment:
When we log messages during project build, we prefix them with "Sonar:". The message will appear in the following form:
```
       SonarWriteProjectData:
         Sonar: The project does not have a valid ProjectGuid. Analysis results for this project will not be uploaded. Project file: D:\src\playgroud\AzureFunctions\AzureFunctions\obj\Debug\net8.0\WorkerExtensions\WorkerExtensions.csproj
         Sonar: (WorkerExtensions.csproj) Project processed successfully
```